### PR TITLE
Adding a method to create an entity naturally

### PR DESCRIPTION
**Pull request by rexbut**
API | [Common](https://github.com/SpongePowered/SpongeCommon/pull/818)

Issue : https://github.com/SpongePowered/SpongeAPI/issues/1107

### CODE REVIEW

**Comment by kenzierocks** [comment]
`spawnEntityNaturally`?

**Comment by rexbut** [comment]
You want to rename functions "createEntityNaturally" ?

File: src/main/java/org/spongepowered/api/world/extent/EntityUniverse.java [hunk 1]
```diff
@@ -150,6 +150,40 @@
...
+     * {@link Optional#empty()} was returned, the entity is not able to spawn at
+     * the given location. Furthermore, this allows for the {@link Entity} to be
+     * customized further prior to traditional "ticking" and processing by core
+     * systems.</p>
+     *
+     * @param type The type
+     * @param position The position
+     * @return An entity, if one was created
+     */
+    Optional<Entity> createNaturallyEntity(EntityType type, Vector3d position);
```
Comment (kashike) [inline src/main/java/org/spongepowered/api/world/extent/EntityUniverse.java]: `createEntityNaturally` sounds better, in my opinion

**Comment by ZephireNZ** [comment]
Correct me if I'm mistaken, but shouldn't this target master? There's no breaking change involved, only an API addition.

**Comment by rexbut** [comment]
Yes I think that could be committed to the master branch

**Comment by gabizou** [comment]
I'd rather add it in API 5.0.0 (bleeding) since there will not likely be any more releases of API 4.x. I'm of the opinion this can be added to bleeding very easily, but @Zidane may have differing opinions.

**Comment by rexbut** [comment]
To fix conflicts, I recreated a new branch : https://github.com/SpongePowered/SpongeAPI/pull/1350

**Comment by ZephireNZ** [comment]
FYI, in the future you can just push to an existing branch to update a PR.

**Pull request closed without merging**